src/external/cute_files.h

```diff
@@ -73,7 +73,7 @@
 
 #define CUTE_FILES_MAX_PATH 1024
 #define CUTE_FILES_MAX_FILENAME 256
-#define CUTE_FILES_MAX_EXT 64
+#define CUTE_FILES_MAX_EXT 32
```

Comment:
Maybe revert this back to 64, IIRC someone had builds with "very very long" extensions and bumped it up to 64

Reply:
Ok